changelog/updates/2022-09-14-intel-microcode.md

```diff
@@ -0,0 +1 @@
+- intel-microcode ([20220809_p20220809](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/releases/tag/microcode-20220809))
```

Comment:
You would then probably want to also add `changelog/security/2022-09-14-intel-microcode-20220809_p20220809.md` that has `intel-microcode ([CVE-2022-21233](https://nvd.nist.gov/vuln/detail/CVE-2022-21233))`.

Reply:
Thanks for pointing this out!